oci8SqlString_test.go

```diff
@@ -184,16 +184,16 @@ func TestSelectDualString(t *testing.T) {
 			results: [][]interface{}{[]interface{}{strings.Repeat("ab", 1000)}},
 		},
 		testQueryResult{
-			args:    []interface{}{"こんにちは 世界 Καλημέρα κόσμε こんにちは안녕하세요góðan dagGrüßgotthyvää päivääyá'át'ééhΓεια σαςВiтаюგამარჯობაनमस्ते你好здравейсвят"},
-			results: [][]interface{}{[]interface{}{"こんにちは 世界 Καλημέρα κόσμε こんにちは안녕하세요góðan dagGrüßgotthyvää päivääyá'át'ééhΓεια σαςВiтаюგამარჯობაनमस्ते你好здравейсвят"}},
+			args:    []interface{}{"こんにちは 世界 Καλημέρα κόσμε こんにちは안녕하세요góðan dagGrüßgotthyvää päivääyá'át'ééhΓεια σαςВiтаюგამარჯობაनमस्ते你好"},
+			results: [][]interface{}{[]interface{}{"こんにちは 世界 Καλημέρα κόσμε こんにちは안녕하세요góðan dagGrüßgotthyvää päivääyá'át'ééhΓεια σαςВiтаюგამარჯობაनमस्ते你好"}},

```

Comment:
I can't read this but is this change expected?

Reply:
Yeah. I can not read it either. Just adding more characters to make sure they work.

Reply:
:+1: